presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/utils/StatementUtils.java

```diff
@@ -132,40 +132,29 @@ private StatementUtils() {}
         builder.put(CreateFunction.class, QueryType.DATA_DEFINITION);
         builder.put(AlterFunction.class, QueryType.DATA_DEFINITION);
         builder.put(DropFunction.class, QueryType.DATA_DEFINITION);
-        builder.put(Use.class, QueryType.DATA_DEFINITION);
-        builder.put(SetSession.class, QueryType.DATA_DEFINITION);
-        builder.put(ResetSession.class, QueryType.DATA_DEFINITION);
-        builder.put(StartTransaction.class, QueryType.DATA_DEFINITION);
-        builder.put(Commit.class, QueryType.DATA_DEFINITION);
-        builder.put(Rollback.class, QueryType.DATA_DEFINITION);
+        builder.put(Use.class, QueryType.TRANSACTION);
+        builder.put(SetSession.class, QueryType.TRANSACTION);
+        builder.put(ResetSession.class, QueryType.TRANSACTION);
+        builder.put(StartTransaction.class, QueryType.TRANSACTION);
+        builder.put(Commit.class, QueryType.TRANSACTION);
+        builder.put(Rollback.class, QueryType.TRANSACTION);
```

Comment:
Just realized that not all these are transactional related. How about calling the query type `CONTROL`?